content/SCALE/SCALEUIReference/Storage/_index.md

```diff
@@ -155,83 +155,83 @@ For example, a system with 18,446,744,073,709,551,615 bytes reports the number a
+ {{< /expand >}}
 
 * {{< expand "Prune Deduplication Table Dialog" "v" >}}
-  The **Prune Deduplication Table** dialog shows pruning measurement options the system should use when pruning the deduplication table (DDT).
-  Options are **Percentage** and **Age**.
-  
-  **Percentage** shows a slider to set the size (maximum amount of storage) percentage threshold the DDT is filled before it reaches the maximum size.
-  When reached the system prunes the table. 
+ The **Prune Deduplication Table** dialog shows pruning measurement options the system should use when pruning the deduplication table (DDT).
+ Options are **Percentage** and **Age**.
+  
+  **Percentage** shows a slider to set the size (maximum amount of storage) percentage threshold the DDT is filled before it reaches the maximum size.
+ When reached the system prunes the table. 
 
-  {{< trueimage src="/images/SCALE/Storage/DedupPruneDialog.png" alt="Prune Deduplication Table - Percentage" id="Prune Deduplication Table - Percentage" >}}
+ {{< trueimage src="/images/SCALE/Storage/DedupPruneDialog.png" alt="Prune Deduplication Table - Percentage" id="Prune Deduplication Table - Percentage" >}}
 
-  **Age** shows the **Age (in days)** field where you enter the number of days between pruning processes the system waits before automatically pruning the DDT.
+  **Age** shows the **Age (in days)** field where you enter the number of days between pruning processes the system waits before automatically pruning the DDT.
 
-  {{< trueimage src="/images/SCALE/Storage/DedupPruneDialogAge.png" alt="Prune Deduplication Table - Age" id="Prune Deduplication Table - Age" >}}
+ {{< trueimage src="/images/SCALE/Storage/DedupPruneDialogAge.png" alt="Prune Deduplication Table - Age" id="Prune Deduplication Table - Age" >}}
 
 {{< /expand >}}
 
 * {{< expand "Deduplication Qutoa Dialog" "v" >}}
-  The **Deduplication Quota for *poolname*** shows the **Quota** dropdown list with three options for setting the maximum size limit the deduplication table can reach.
-  **Auto** is the default option, which allows the system to set the quota and the size of a dedicated dedup VDEV. This property works for both legacy and fast dedup tables.
-  
-  {{< trueimage src="/images/SCALE/Storage/DedupQutoaDialog.png" alt="Deduplication Qutoa for Pool Dialog" id="Prune Deduplication Table Dialog" >}}
+ The **Deduplication Quota for *poolname*** shows the **Quota** dropdown list with three options for setting the maximum size limit the deduplication table can reach.
+  **Auto** is the default option, which allows the system to set the quota and the size of a dedicated dedup VDEV. This property works for both legacy and fast dedup tables.
+  
+ {{< trueimage src="/images/SCALE/Storage/DedupQutoaDialog.png" alt="Deduplication Qutoa for Pool Dialog" id="Prune Deduplication Table Dialog" >}}
```

Comment:
Same thing here, kind of an odd spacing in the local build test: 
![image](https://github.com/user-attachments/assets/57e9fd62-60a9-46a7-87ac-ccff67d906d0)

```suggestion
 The **Deduplication Quota for *poolname*** shows the **Quota** dropdown list with three options for setting the maximum size limit the deduplication table can reach.

 **Auto** is the default option, which allows the system to set the quota and the size of a dedicated dedup VDEV. This property works for both legacy and fast dedup tables.
  
 {{< trueimage src="/images/SCALE/Storage/DedupQutoaDialog.png" alt="Deduplication Qutoa for Pool Dialog" id="Prune Deduplication Table Dialog" >}}
```